<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>418</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Inicio</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-30</y>
     <width>1131</width>
     <height>411</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <widget class="QLabel" name="lbImgItem1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbTextItem1">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Regresión Lineal</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnItem1">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget2" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <widget class="QLabel" name="lbImgItem2">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbTextItem2">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tabla de Datos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnItem2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>220</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget3" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <widget class="QLabel" name="lbImgItem3">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbTextItem3">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Productos</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnItem3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget4" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <widget class="QLabel" name="lbImgItem4">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbTextItem4">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Red neuronal Classificación</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnItem4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_11">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QWidget" name="widget5" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>300</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <widget class="QLabel" name="lbImgItem5">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>120</width>
         <height>120</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="lbTextItem5">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>150</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Red neuronal Regresión Lineal</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="btnItem5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>220</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ver</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
